docs/src/reference/creds.md

```diff
@@ -0,0 +1,229 @@
+# Client Credentials
+
+Greenlight utilizes various components, such as mTLS certificates and 
+runes, to provide secure access to a node. 
+Various pieces of data must be persisted and made available to the client 
+at runtime. Check out the [Security][security] page for more information.
+
+To simplify data management for users and developers, Greenlight uses 
+`Credentials` as a centralized location for the necessary information. 
+Credentials can be reconstructed in various ways and exported in 
+byte-encoded format for persistence.
+
```

Comment:
Provide an introduction to the different type of credentials and when you want to use them.

Suggestion

```
The nobody-credential is not linked to a specific node. You can use this credential to `register` a new node or `recover` an existing one. We don't allow everyone to spin up an unlimited number of nodes. You can download your developer certificate on `https://greenlight.blockstream.com`.

A device credential can be used to connect and authenticate a device. You receive device-credentials when you `register` or `recover` a node. These credentials can be stored on disk or encoded as binary data.
```